treeherder/services/signatures_remover.py

```diff
@@ -0,0 +1,146 @@
+from typing import List
+
+from treeherder.perf.models import PerformanceDatum, PerformanceSignature
+from treeherder.services.taskcluster import TaskclusterModel
+from treeherder.services.max_runtime import MaxRuntime
+
+RECEIVER_TEAM_EMAIL = "perftest-alerts@mozilla.com"
+EMAIL_RECEIVER = "dhunt@mozilla.com"
+
+
+class PublicSignaturesRemover:
+    """
+    This class handles the removal of signatures which are (no longer)
+    associated with a job and sends email notifications to the entire team.
+    """
+
+    html_content = """
+        <html>
+            <head>
+            </head>
+            <body>
+                <h2>Summary of deleted Performance Signatures</h2>
+                <table>
+                    <tr>
+                        <th>repository</th>
+                        <th>framework</th>
+                        <th>platform</th>
+                        <th>suite</th>
+                        <th>application</th>
+                    </tr>
+    """
+
+    end_of_html_content = """
+                </table>
+            </body>
+        </html>
+    """
+
+    def __init__(
+        self,
+        timer: MaxRuntime,
+        taskcluster_model: TaskclusterModel,
+    ):
+        self.tc_model = taskcluster_model
+        self._subject = "Summary of deleted Performance Signatures"
+        self._content = None
+        self._nr_of_rows_allowed_in_email = 10  # TO BE DEFINED
+        self._nr_of_emails_allowed = 4  # TO BE DEFINED
+
+        self.timer = timer
+
+    def remove(self):
+        nr_of_emails_sent_so_far = 0
+
+        nr_of_rows_left_to_complete_email = self._nr_of_rows_allowed_in_email
```

Comment:
These variable names are very clear, but they're just too long. Let's look for more concise names.